lib/DateTime.php

```diff
@@ -159,13 +159,13 @@ public function setDate($year, $month, $day)
 		return parent::setDate($year, $month, $day);
 	}
 
-	public function setISODate($year, $week , $day = 1)
+	public function setISODate($year, $week, $day = 1)
 	{
 		$this->flag_dirty();
 		return parent::setISODate($year, $week, $day);
 	}
 
-	public function setTime($hour, $minute, $second = 0)
+	public function setTime($hour, $minute, $second = null, $microseconds = null)
 	{
 		$this->flag_dirty();
 		return parent::setTime($hour, $minute, $second);
```

Comment:
Shouldn't the argument be passed through?

Reply:
Muchas gracias, funciono para mi